msgpack-core/src/main/java/org/msgpack/core/MessageUnpacker.java

```diff
@@ -1261,12 +1491,35 @@ public void readPayload(ByteBuffer dst)
         }
     }
 
+    /**
+     * Reads payload bytes of binary, extension, or raw string types.
+     *
+     * This consumes specified amount of bytes into the specified byte array.
+     *
+     * <p>
+     * This method is equivalent to <code>readPayload(dst, 0, dst.length)</code>.
+     *
+     * @param dst the byte array into which the data is read
+     * @throws IOException when underlaying input throws IOException
+     */
     public void readPayload(byte[] dst)
             throws IOException
     {
         readPayload(dst, 0, dst.length);
```

Comment:
I think people will use `readPayload(int)` for that use case. It's already there. So, I think it's ok to return void here. Having consistency with `readPayload(byte[], int off, int len)` might be more important to avoid confusion.